src/entities/stabilityTracker/ui/StabilityTrackerItem.tsx

```diff
@@ -348,19 +344,17 @@ const StabilityTrackerItemScreen = (props: Props) => {
       lambdaSlope: lambdaSlope.current,
     };
 
-    if (shouldSendLiveEvent) {
-      const liveEvent: StabilityTrackerAnswerValue = {
-        timestamp: response.timestamp,
-        stimPos: response.circlePosition,
-        targetPos: response.targetPosition,
-        userPos: response.userPosition,
-        score: response.score,
-        lambda: response.lambda,
-        lambdaSlope: response.lambdaSlope,
-      };
-
-      sendLiveEvent(liveEvent);
-    }
+    const liveEvent: StabilityTrackerAnswerValue = {
+      timestamp: response.timestamp,
+      stimPos: response.circlePosition,
+      targetPos: response.targetPosition,
+      userPos: response.userPosition,
+      score: response.score,
+      lambda: response.lambda,
+      lambdaSlope: response.lambdaSlope,
+    };
+
+    sendLiveEvent(liveEvent);
```

Comment:
What happens if the Live Connection has been turned on one applet but then the user goes to play with Stability Tracker on the other applet that does not support the Live Connection?